pyqir-jit/src/python.rs

```diff
@@ -2,10 +2,15 @@
 // Licensed under the MIT License.
 
 use crate::{interop::Instruction, jit::run_module_file};
-use pyo3::{exceptions::PyOSError, prelude::*, types::PyDict};
+use bitvec::prelude::BitVec;
+use pyo3::{
+    exceptions::PyOSError,
+    prelude::*,
+    types::{PyDict, PyList},
+};
 
 #[pymodule]
-fn pyqir_jit(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
+fn _native(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
```

Comment:
Is the `_` here somehow because the python interop doesn't let rust see that this is actually used, or is this function really not used anywhere? If the latter, can it just be removed?

Reply:
If I understand correctly, this is to mark it as private to the Python module once it's imported there. That helps address issues like #29, where Rust implementation details can inadvertently become visible to pure-Python callers.

Reply:
It is a requirement that the name of the pymodule init function match the name of the pyd native module, which is `_native.py`. By default, `#[pymodule]` uses the name of the Rust function it's attached to for the init function.